core/transformer/src/IModelImporter.ts

```diff
@@ -15,6 +15,16 @@ import { ElementAspect, ElementMultiAspect, Entity, IModelDb, Model, Relationshi
 
 const loggerCategory: string = TransformerLoggerCategory.IModelImporter;
 
+/** Options provided to [[IModelImporter.optimizeGeometry]] specifying post-processing optimizations to be applied to the iModel's geometry.
+ * @beta
+ */
+export interface OptimizeGeometryOptions {
```

Comment:
How does this and IModelImportOptions.simplifyElementGeometry relate?  Should setting simplifyElementGeometry turn on all geometry optimizations?

Reply:
That makes sense.  This should be documented so it is clear to callers that the two options are independent.  @MichaelBelousov

Reply:
I will add better documentation on how they are related in a separate PR